include/swift/AST/ASTBridging.h

```diff
@@ -93,6 +93,24 @@ struct BridgedLocatedIdentifier {
   BridgedSourceLoc NameLoc;
 };
 
+struct BridgedConsumedLookupResult {
+  SWIFT_NAME("name")
+  BridgedIdentifier Name;
+
+  SWIFT_NAME("nameLoc")
+  BridgedSourceLoc NameLoc;
+
+  SWIFT_NAME("flag")
+  SwiftInt Flag;
+
+#ifdef USED_IN_CPP_SOURCE
+  BridgedConsumedLookupResult(swift::Identifier name,
```

Comment:
This will trip us up with Visual C++ because the calling convention for a class with a user-defined constructor is different from one without. I think you can add a dummy constructor outside of the `USED_IN_CPP_SOURCE` to get consistent behavior, e.g.,

```
BridgedConsumedLookupResult(void*, void*, int);
```

Reply:
Even an empty default constructor will work, that's probably better:

```
BridgedConsumedLookupResult() { }
```

Reply:
The ideal way of doing this would be using `BRIDGED_INLINE` by declaring the constructor here and defining it in ASTBridgingImpl.h, see e.g the BridgedDeclBaseName constructor below. This avoids the need for `USED_IN_CPP_SOURCE`.

Reply:
Thank you for the suggestion @hamishknight! I removed the dummy constructor with `USED_IN_CPP_SOURCE` and moved the definition to `ASTBridgingImpl.h`.